x/subspaces/simulation/genesis.go

```diff
@@ -23,7 +23,7 @@ func RandomizeGenState(simState *module.SimulationState) {
 	members := randomUserGroupsMembers(simState.Rand, simState.Accounts, groups)
 	acl := randomACL(simState.Rand, simState.Accounts, subspaces)
 	initialSubspaceID, subspacesData := getSubspacesDataEntries(subspaces, sections, groups)
-	grants := append(randomUserGrants(simState.Rand, simState.Accounts, subspaces), randomGroupGrants(simState.Rand, simState.Accounts, groups)...)
+	grants := append(randomUserGrants(simState.Rand, simState.Accounts, subspaces), randomGroupGrants(simState.Rand, groups)...)
```

Comment:
Why was the code inside this simulation changed?

Reply:
The previous simulation test is failed since the behaviour was wrong. The granter must be a subspace treasury instead of a random user address.